hack/make-rules/verify.sh

```diff
@@ -69,11 +78,17 @@ function is-quick {
 }
 
 function run-cmd {
+  set +o nounset
+  local filename="${2##*/verify-}"
+  local testname="${filename%%.*}"
+  local output="${KUBE_JUNIT_REPORT_DIR:-/tmp/junit-results}"
+
   if ${SILENT}; then
```

Comment:
Always prefer to be explicit

```sh
if [[ -n "${SILENT:-}" ]]; then
```

Or even better, check for the specific string you expect to be passed